src/components/SEP/SEPStatusFilter.tsx

```diff
@@ -0,0 +1,55 @@
+import FormControl from '@material-ui/core/FormControl';
```

Comment:
As far as I know we have this similar active/inactive filter for calls but not sure if that can be reused or not.